Building documentation failed: No module named 'sphinx_copybutton'

**Snakemake version**
Snakemake 5.26.1 and snakemake-wrappers 0.66.0
**Describe the bug**
Building the documentation failed with an extension error `No module named 'sphinx_copybutton'`.
**Logs**
```
Extension error:
L'extension ne peut pas être importée sphinx_copybutton (exception: No module named 'sphinx_copybutton')
Makefile:20 : la recette pour la cible « html » a échouée
make: *** [html] Erreur 2
```
**Minimal example**
```
conda create -n test-snakemake-wrapper-docs sphinx sphinx_rtd_theme pyyaml
conda activate test-snakemake-wrapper-docs
cd docs
make html
```
**Additional context**
The code was executed within the `test-snakemake-wrapper-docs` environment as documented on the _Testing locally_ section [here](https://snakemake-wrappers.readthedocs.io/en/stable/index.html)

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -86,7 +86,7 @@ the relevant stdout and stderr messages printed.
 If you also want to test the docs generation locally, create another environment
 and activate it::
 
-  conda create -n test-snakemake-wrapper-docs sphinx sphinx_rtd_theme pyyaml
+  conda create -n test-snakemake-wrapper-docs sphinx sphinx_rtd_theme pyyaml sphinx-copybutton
   conda activate test-snakemake-wrapper-docs
 
 Then, enter the respective directory and build the docs::

